frontend/src/pages/org/archived-item-detail/archived-item-detail.ts

```diff
@@ -683,6 +683,7 @@ export class ArchivedItemDetail extends TailwindElement {
           ? html`<div id="replay-crawl" class="aspect-4/3 overflow-hidden">
               <replay-web-page
                 source="${replaySource}"
+                url="${this.crawl.seedCount === 1 ? this.crawl.firstSeed : ""}"
```

Comment:
Looks like this is throwing a linting error from lit-plugin: `This attribute binds the type 'string | null' which can end up binding the string 'null'. Use the 'ifDefined' directive?` do you see this warning in your IDE? It can be fixed with `ifDefined(this.crawl.firstSeed)`.

Or, if we want this to only apply for seeded crawled, instead of any crawl with one URL, you can check `this.workflow?.jobType`:

```suggestion
                url="${this.workflow?.jobType === 'seed-crawl' && this.crawl.firstSeed || ""}"
```

Reply:
Didn't have lit-plugin enabled, looks like I'll have to get used to that syntax highlighting 😅 

I considered using the workflow type, but wanted this to also work for URL list crawls with only 1 URL.

Reply:
enabled lit-plugin, also added the url attribute to `replayWebPage.d.ts` as a string, which causes the type to be upset with that expression so I added a string conversion and now everything is happy?